loading/data/splign-manual/generate-loading-data

```diff
@@ -79,6 +80,7 @@ def parse_splign(fn, txdata):
     txinfo = uta.formats.txinfo.TxInfo(
         origin=origin,
         ac=tx_ac,
+        gene_id=gene_id,
         hgnc=hgnc,
```

Comment:
For `TxInfo` we renamed "hgnc" to "gene_symbol". Do we need to update this line?